phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewMetadataIT.java

```diff
@@ -1173,10 +1174,10 @@ private void helpTestQueryForViewOnTableThatHasIndex(Statement s1,
         s1.execute("create index " + indexName + " ON " + tableName
                 + " (col2)");
 
-        try (ResultSet rs = s2.executeQuery("explain select /*+ INDEX("
+        try (ResultSet rs = s2.executeQuery("select /*+ INDEX("
```

Comment:
why this change?

Reply:
will look into this

Reply:
Fixed the problem in https://issues.apache.org/jira/browse/PHOENIX-7307
Will revert this change

Reply:
reverted